### PR TITLE
Add week five schemes

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: code_schemes/icraf_s01e05.json [hunk 1]
```diff
@@ -2,70 +2,181 @@
...
+            "StringValue": "use_compost_manure",
+            "DisplayText": "use compost manure",
+            "VisibleInCoda": true,
+            "NumericValue": 1,
+            "CodeID": "code-1ccbeb31",
+            "CodeType": "Normal"
+        },
+        {
+            "StringValue": "farmers_knowledge",
+            "DisplayText": "farmers knowledge",
```
Comment (as2388) [inline code_schemes/icraf_s01e05.json]:
```suggestion
            "DisplayText": "farmers' knowledge",
```

File: code_schemes/icraf_s01e05.json [hunk 2]
```diff
@@ -2,70 +2,181 @@
...
+        {
+            "StringValue": "tree_planting",
+            "DisplayText": "tree planting",
+            "VisibleInCoda": true,
+            "NumericValue": 8,
+            "CodeID": "code-e2f899f1",
+            "CodeType": "Normal"
+        },
+        {
+            "StringValue": "certfied_seeds",
```
Comment (as2388) [inline code_schemes/icraf_s01e05.json]:
```suggestion
            "StringValue": "certified_seeds",
```

**Review by as2388 — APPROVED** [review]

File: code_schemes/icraf_s01e05.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/icraf_s01e05.json]: updated thanks!

File: code_schemes/icraf_s01e05.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/icraf_s01e05.json]: Updated thanks!

**Pull request merged**